Remove footnotes from the language reference guide (#847)
Dyalog/documentation: language-reference-guide/docs/system-functions/na.md

Reviewer: ```suggestion
- a Dynamic Link Library (**.dll**) under Microsoft Windows
```

>     - a Shared Library (.so or .dylib) under Linux or macOS

Reviewer: ```suggestion
- a Shared Library (**.so** or **.dylib**) under Linux or macOS
```

>     - a static library (.a) under AIX

Reviewer: ```suggestion
- a static library (**.a**) under AIX
```

> A library, refered to here generically as a DLL, is a collection of functions typically written in C (or C++) each of which may take arguments and return a result.

Reviewer: ```suggestion
A library, referred to here generically as a Dynamic Link Library (DLL), is a collection of functions typically written in C (or C++) each of which can take arguments and return a result.
```

|Type       |`I`     |int                                                                                                    |
|           |`U`     |unsigned int                                                                                           |
|           |`C`     |char                                                                                                   |

> |           |`T`     |char                                                                                                   |

Reviewer: ```suggestion
|           |`T`     |char (Classic Edition: translated to/from ANSI)                                                                                                 |
```

|           |`F`     |float                                                                                                  |
|           |`D`     |decimal                                                                                                |
|           |`J`     |complex                                                                                                |

> |           |`P`     |uintptr-t (equivalent to U4 on 32-bit versions and U8 on 64-bit)                                       |

Reviewer: ```suggestion
|           |`P`     |uintptr-t (equivalent to U4 and U8 on 32-bit and 64-bit widths respectively)                                       |
```
